feat: implement condition 9 #11

Storsorken opened the pull request:
Closes #11

Storsorken committed:
feat: implement condition 9 #11

paulinegarelli committed:
Merge branch 'main' into erik/feat/11-add-condition9

paulinegarelli reviewed (changes requested):
Same problem here as for condition 2, you mixed the two cases and one of them is missing!

Also, just thought about this but maybe we should change the angle function so that if two points coincide, we return some kind of "no angle" statement or a null value.

Storsorken committed:
Merge branch 'main' into erik/feat/11-add-condition9

Storsorken committed:
fix: bug in condition9 #53 @paulinegarelli

Method did not check both cases for the angle or that the angle could be invalid as described in #11

Storsorken committed:
Merge branch 'main' into erik/feat/11-add-condition9

hampfh reviewed (approved):
👍
